test/files/run/duration_negative.scala

```diff
@@ -0,0 +1,14 @@
+import scala.concurrent.duration._
+
+object Test extends App {
+  // Calling Duration.apply with 1L and 1D should 
+  // result in the same Duration
+  assert(Duration(-1, DAYS) == Duration(-1.0, DAYS))
+  
+  // A Duration of -10 seconds should be -10000000000 long
+  assert(Duration("-10 s").toNanos == -10000000000l)
+  
+  // Positive Durations should still work
+  assert(Duration(1, DAYS) == Duration(1.0, DAYS))
+  assert(Duration("10 s").toNanos == 10000000000L)
+}
```

Comment:
Sure. I wasn't sure where to test it and this was the only place I found other tests for Duration.